build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -1061,15 +1060,14 @@ CONFIG_MTD_SMART_WEAR_LEVEL=y
 # Memory Management
 #
 CONFIG_MM_KERNEL_HEAP=y
-CONFIG_MM_KERNEL_HEAPSIZE=2097152
+CONFIG_MM_KERNEL_HEAPSIZE=33554432
```

Comment:
I think we can merge `CONFIG_MM_KERNEL_HEAPSIZE` and `CONFIG_RAM_KREGIONx_SIZE`

Reply:
Even though `CONFIG_MM_KERNEL_HEAPSIZE ` is not used in imxrt project, it is used in some other projects. If we remove it, then we need to change the code in those places. 

Also,` CONFIG_MM_KERNEL_HEAPSIZE` indicates a single value, but `CONFIG_RAM_KREGIONx_SIZE` maybe an array of values if we have more than one ram regions. So, I am not sure how we can merge them.

Reply:
I agree that if we merge those two, then we should change the code which using CONFIG_MM_KERNEL_HEAPSIZE.
But we are already using `regionx_start` and `regionx_size` when we have one user heap, So I think it is not awkward.
What do you think about this @kishore-sn  and @sunghan-chang ?

Reply:
Ok. I think it can be done. The `CONFIG_MM_KERNEL_HEAPSIZE` config seems to be mostly unused. So, I have removed it and added the changes in a new commit.

Reply:
Why we suggest and make this PR is unifying heap management including configuration.
Whatever we use, we should provide one way to do it. 
If this represents all of kernel heap size with several kernel heap regions, we could calculate this automatically.
```
CONFIG_RAM_KREGIONx_SIZE="10,20"

CONFIG_MM_KERNEL_HEAPSIZE = 10 + 20 = 30
```